29.0.1 (#825)
MetaMask/controllers: CHANGELOG.md

## [Unreleased]

## [29.0.1]
### Added

> - Add utils to format token list tokens for aggregators and icon url. ([#810](https://github.com/MetaMask/controllers/pull/810))

Reviewer: This is an internal-only change. It's a new method in a module that is not exported.
```suggestion
```

> - Add SupportedTokenDetectionNetworks enum and util for token detection support. ([#811](https://github.com/MetaMask/controllers/pull/811))

Reviewer: Hmm.... usually we use a minor bump for releases for additions.

I guess we can let it slide this time, since this isn't meant to be an externally-facing change, and we're in a hurry. Unfortunate that all of our util functions are being exported right now, even those that are only used internally.

Reviewer 2: yeah I was wondering about this. But also figured we'd decided to not revert these commits so

### Fixed

> - Fix bug introduced in v29.0.0 where serialized seedphrases passed to `createNewVaultAndRestore` were not correctly validated. ([#819](https://github.com/MetaMask/controllers/pull/819))

Reviewer: Nit: This makes it sound like it was allowing invalid seeds, or something like that. Maybe clearer to say it was crashing.
```suggestion
- Fix bug introduced in v29.0.0 where `createNewVaultAndRestore` would crash with a validation failure. ([#819](https://github.com/MetaMask/controllers/pull/819))
```
